errors/no-img-element.mdx

```diff
@@ -68,8 +68,20 @@ function Home() {
 }
 ```
 
+4. You can use a [custom image loader](/docs/pages/building-your-application/optimizing/images#loaders) to optimize images. Set [loaderFile](/docs/pages/api-reference/components/image#loaderfile) to the path of your custom loader.
+
+```js filename="next.config.js"
```

Comment:
Should we also include a `next.config.ts` version?

Reply:
We could follow up!